test/bookshop/package.json

```diff
@@ -48,6 +48,10 @@
       "_outbox": {
         "kind": "persistent-outbox"
       },
+      "queue": {
+        "kind": "persistent-queue",
```

Comment:
```suggestion
```

Reply:
use the default of the respective cds version

Reply:
actually, cds8 doesn't know `queue`, so shouldn't make a difference. nevertheless, we shouldn't set default values here.